### ISSUE TITLE
A notification bell appears for a 'tenth of second' next to the searchbox

### ISSUE THREAD

**Issue by davitol** [owncloud/core#19729]
### Steps to reproduce

1. Log in oC server
2. Refresh the browser

### Expected behaviour

The page should be refreshed with the searchbox

### Actual behaviour

A notification bell appears for a 'tenth of second'

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** ownCloud 8.2 RC1 (daily) Build:2015-10-13T03:37:34+00:00 b9e515bfeb934efa98beda9e9aa38d232933aba9

**Updated from an older ownCloud or fresh install:**
Fresh

**List of activated apps:** LDAP

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
No

**Logs**
```
```
### Client configuration

**browser**
 Firefox.

![screen shot 2015-10-13 at 09 45 55](https://cloud.githubusercontent.com/assets/10596560/10448840/df7e6f90-718f-11e5-8728-9c99a9962607.png)

**Comment by PVince81** [comment]
@nickvergessen you could make it hidden by default and only display it after the call returns, the call that decises whether it needs to be shown or not.

But then someone will raise a ticket called "notification bell doesn't appear directly and only after a specific time"

**Comment by nickvergessen** [comment]
In 9.0 we will have an app there by default, so the bell will not be hidden. Until then I'd say this is not worth fixing.
It's like displaying a loading spinner for the tenth of a second on the preview in the sidebar and stuff like that.

**Comment by PVince81** [comment]
@nickvergessen @davitol close / won't fix ?

**Comment by nickvergessen** [comment]
I'd say yes

**Comment by davitol** [comment]
:+1:

**Comment by SergioBertolinSG** [comment]
Still happening in master and 8.2.1rc1 (enterprise theme).
@nickvergessen @PVince81 not worth to fix it for 8.2.1 as well?

**Comment by nickvergessen** [comment]
9.0 as said
